### DISCUSSION TITLE
Centralized Pumping

### DISCUSSION THREAD

**Discussion by TBN-MapleWheels** [PeasantFoundry/Mechtrauma#44] [Ideas]
Originally posted by @PeasantFoundry 

Allows for a centralized water pump to operate in multiple locations at once by using drains.

Details:

**Comment by PanmanS** [comment]
isnt it already W.I.P


Anyways i have a working protype build on base of berillia

**Comment by Torca2001** [comment]
Centralized pumps respect the requirement for water gates that other Mechtrauma pumps require.

My latest PR implements water gates with special connections to prevent cheesing.

**Comment by TBN-MapleWheels** [comment]
Currently in PR #64 .

**Comment by Torca2001** [comment]
On request from Hadrada these pumps have a check box (default off) for whether the pump will require a water gate so this can be decided by the sub designer
